code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -0,0 +1,335 @@
+	name = "Salicyclic Acid"
+	id = "sal_acid"
+	result = "sal_acid"
+	required_reagents = list("sodium" = 1, "phenol" = 1, "carbon" = 1, "oxygen" = 1, "sacid" = 1)
+	result_amount = 5
+
+datum/reagent/salbutamol
+	name = "Salbutamol"
+	id = "salbutamol"
+	description = "Heals 6 OXY damage, reduces LOSEBREATH by 4."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.2
+
+datum/reagent/salbutamol/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustOxyLoss(-6*REM)
+	if(M.losebreath >= 4)
+		M.losebreath -= 4
+	..()
+	return
+
+/datum/chemical_reaction/salbutamol
+	name = "Salbutamol"
+	id = "salbutamol"
+	result = "salbutamol"
+	required_reagents = list("sal_acid" = 1, "lithium" = 1, "aluminium" = 1, "bromine" = 1, "ammonia" = 1)
+	result_amount = 5
+
+datum/reagent/perfluorodecalin
+	name = "Perfluorodecalin"
+	id = "perfluorodecalin"
+	description = "Heals 25 OXY damage, but you can't talk. 33% chance of healing 1 BRUTE and 1 BURN."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.2
+
+datum/reagent/perfluorodecalin/on_mob_life(var/mob/living/carbon/human/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustOxyLoss(-25*REM)
+	M.silent = max(M.silent, 5)
+	if(prob(33))
+		M.adjustBruteLoss(-1*REM)
+		M.adjustFireLoss(-1*REM)
+	..()
+	return
+
+/datum/chemical_reaction/perfluorodecalin
+	name = "Perfluorodecalin"
+	id = "perfluorodecalin"
+	result = "perfluorodecalin"
+	required_reagents = list("hydrogen" = 1, "fluorine" = 1, "oil" = 1)
+	result_amount = 3
+	required_temp = 370
+	mix_message = "The mixture rapidly turns into a dense pink liquid."
+
+datum/reagent/ephedrine
+	name = "Ephedrine"
+	id = "ephedrine"
+	description = "Stun reduction per cycle, increases run speed slightly, minor stamina regeneration buff, stabilizes crit."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 0.3
+
+datum/reagent/ephedrine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.status_flags |= GOTTAGOFAST
+	if(M.losebreath >= 10)
+		M.losebreath = max(10, M.losebreath-5)
+	M.AdjustParalysis(-1)
+	M.AdjustStunned(-1)
+	M.AdjustWeakened(-1)
+	M.adjustStaminaLoss(-1*REM)
+	..()
+	return
+
+/datum/chemical_reaction/ephedrine
+	name = "Ephedrine"
+	id = "ephedrine"
+	result = "ephedrine"
+	required_reagents = list("sugar" = 1, "oil" = 1, "hydrogen" = 1, "diethylamine" = 1)
+	result_amount = 4
+	mix_message = "The solution fizzes and gives off toxic fumes."
+
+datum/reagent/diphenhydramine
+	name = "Diphenhydramine"
+	id = "diphenhydramine"
+	description = "Stun reduction per cycle, increases run speed slightly, minor stamina regeneration buff, stabilizes crit."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+datum/reagent/diphenhydramine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.drowsyness -= 1
+	M.jitteriness -= 1
+	..()
+	return
+
```

Comment:
Diphenhydramine's biggest selling point is that that is purges histamine; it increases the metabolization_rate of histamine by 3.

Also, it causes drowsyness, not reduces it =p